packages/netlify/src/ssr-function.ts

```diff
@@ -24,7 +25,13 @@ export const createExports = (manifest: SSRManifest, { middlewareSecret }: Args)
 		cacheOnDemandPages: boolean;
 		notFoundContent?: string;
 	}) {
+		// The entrypoint is created in `.netlify/v1/build` so we need to go up two levels to get to the root
+		const root = fileURLToPath(new URL('../../', import.meta.url));
 		return async function handler(request: Request, context: Context) {
+			// This entrypoint will be deep inside the directory structure below cwd
+			// We chdir to the root so that we can resolve the correct paths at runtime
+			process.chdir(root);
```

Comment:
`new URL` and `process.chdir` are likely to fail, I would play it a bit more conservative: use `URL.canParse` to check if it's a valid a URL, and a `try/catch` for `process.chdir` to at least not fail (or fail) and gracefully log the error. Even Node.js docs recommend using the `try/catch` block https://nodejs.org/api/process.html#processchdirdirectory

Reply:
What are you suggesting we check with `URL.canParse`? `import.meta.url`?

Reply:
The whole thing I suppose: 

```js
if (URL.canParse("../../", import.meta.url)) {}
```

Reply:
What would be the scenario where it wouldn't work though, when `"../.."` is a static value, and `import.meta.url` comes from the runtime?

Reply:
There was a bug in the Cloudflare adapter where `import.meta.url` was throwing an error because it wasn't a URL. I am not saying that Netlify will get wrong, just playing it safe 😅

Reply:
If `import.meta.url` is broken I don't think we should hide that!

Reply:
I've added a try/catch, but don't think we need the URL check

Reply:
I think the issue with the Cloudflare adapter was not that `import.meta.url` wasn't an URL, I think it is just not supported in Cloudflare's runtime, therefore not working for `new URL()`